tipg/main.py

```diff
@@ -44,8 +48,190 @@
 )  # type: ignore
 
 # Register endpoints.
-endpoints = Endpoints(title=settings.name, templates=templates)
-app.include_router(endpoints.router, tags=["OGC API"])
+ogc_features = OGCFeaturesFactory(templates=templates)
+app.include_router(ogc_features.router, tags=["OGC Features API"])
+
+ogc_tiles = OGCTilesFactory(templates=templates)
+app.include_router(ogc_tiles.router, tags=["OGC Tiles API"])
+
+
+@app.get(
+    "/conformance",
+    response_model=Conformance,
+    response_model_exclude_none=True,
+    response_class=ORJSONResponse,
+    responses={
+        200: {
+            "content": {
+                MediaType.json.value: {},
+                MediaType.html.value: {},
+            }
+        },
+    },
+)
+def conformance(
+    request: Request, output_type: Optional[MediaType] = Depends(OutputType)
+):
+    """Get conformance."""
+    data = Conformance(
+        conformsTo=[
+            # OGC Common
+            "http://www.opengis.net/spec/ogcapi-common-1/1.0/conf/core",
+            "http://www.opengis.net/spec/ogcapi-common-1/1.0/conf/landingPage",
+            "http://www.opengis.net/spec/ogcapi-common-2/1.0/conf/collections",
+            "http://www.opengis.net/spec/ogcapi-common-2/1.0/conf/simple-query",
+            "http://www.opengis.net/spec/ogcapi-common-1/1.0/conf/json",
+            "http://www.opengis.net/spec/ogcapi-common-1/1.0/conf/html",
+            "http://www.opengis.net/spec/ogcapi-common-1/1.0/conf/oas30",
+            *ogc_features.conforms_to,
+            *ogc_tiles.conforms_to,
+        ]
+    )
+
+    if output_type == MediaType.html:
+        return create_html_response(
+            request,
+            data.json(exclude_none=True),
+            templates=templates,
+            template_name="conformance",
+        )
+
+    return data
```

Comment:
We hard code both conformance and landing but there is a method in the OGCFeaturesFactory if a user wants to create a standalone OGC Features API (with all the required endpoints)

Reply:
should we just have a single conformance/landing factory instead?